ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh

```diff
@@ -18,7 +18,10 @@ GSHEET_KEY_LOCATION="/ga-gsheet/gcp-sa-account"
 export GSHEET_KEY_LOCATION
 
 git clone https://github.com/cloud-bulldozer/e2e-benchmarking
-pushd e2e-benchmarking/workloads/kube-burner
+pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
+#export EXTRA_FLAGS="--pods-per-node=$PODS_PER_NODE "
+#export NAMESPACED_ITERATIONS=${NAMESPACED_ITERATIONS:-false}
+#export EXTRA_FLAGS=${EXTRA_FLAGS:-}
```

Comment:
These variables can be deleted, as they are referenced in [openshift-qe-node-density-heavy-ref.yaml](https://github.com/openshift/release/pull/41745/files#diff-fd8637f16f5b639fef15ec39022b495a5a84001e7dbf87407f22cceb3dbbbdc5)

Reply:
I plan to remove them once tests pass Harsha. Not sure why I added PODS_PER_NODE to EXTRA_FLAGS before. So, wanted to make sure tests run successfully before I delete it.